configs/next_router_vuejs.json

```diff
@@ -19,7 +19,7 @@
     }
   ],
   "stop_urls": [
-    ".html"
+    "(?<!\\.html|/)$"
```

Comment:
I don't know what is used behind the scenes but it requires a negative look behind and I escaped the `\` so it can escape the `.`. But I also don't know if there is a special syntax to say this is a regex

Reply:
We use Python so we will need fixed-width look behind, this should do it https://github.com/algolia/docsearch-configs/pull/3896/commits/2c3b5c04ce3268a6ce0546128fb18afd179b73fb

if there's any other indices feel free to open the PR/let me know :D